Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: If, at the time fixed for the beginning of the term of the President, the President elect shall have died, the Vice President elect shall become President. If a President shall not have been chosen before the time fixed for the beginning of his term, or if the President elect shall have failed to qualify, then the Vice President elect shall act as President until a President shall have qualified; and the Congress may by law provide for the case wherein neither a President elect nor a Vice President elect shall have qualified, declaring who shall then act as President, or the manner in which one who is to act shall be selected, and such person shall act accordingly until a President or Vice President shall have qualified.   affects 9    affects 14
+4: It is the Right of all People to petition their Government for redress of grievances; and the Responsibility of all People to watch their Government and ensure its good behavior, and to attempt to peaceably resolve intrusions by Government on the Rights of the People, before rising in arms against such an intrusion; and to rise in arms against the Government when peaceful resolutions become impossible.
 
-4: The Congress may by law provide for the case of the death of any of the persons from whom the House of Representatives may choose a President whenever the right of choice shall have devolved upon them, and for the case of the death of any of the persons from whom the Senate may choose a Vice President whenever the right of choice shall have devolved upon them.   affects 9
+5: It is the Right of all People to practice their religion freely and openly; and the Responsibility of all People to avoid interfering in the religious practice of others, and, in the practice of their religion, to respect the Rights of other People.
 
-5: Sections 1 and 2 shall take effect on the 15th day of October following the ratification of this article.
+6: It is the Right of all People to be considered innocent of any crime, unless such crime shall be proven in a court of law, as decided by unanimous decision of a jury of at least twelve disinterested peers, and for such a trial to be carried out with reasonable speed in public, and for the result of such a trial to be respected in perpetuity; and to have access to legal Representation provided by the court or the prosecution, when the accused Person is unable to retain such Representation by their own means; and to be reasonably compensated for time served on a jury; and the Responsibility of all People to serve on such juries when called upon, and in such service adhere to the Truth, the Rights of the People, and the Law; and in such cases as the Law is in conflict with Truth or the Rights of the People, to judge in favor of Truth and the Rights of the People, against that unjust Law.
```

Comment:
Here I lay out the clear distinction between a right to receive the services of a lawyer, and the right not to be convicted without having access to the services of a lawyer.